portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,23 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+fetch('/data').then(response => response.json()).then((comment) => {
```

Comment:
The `comment` variable appears to be a list of comments, but the name implies that it is one comment. Let's rename it in the plural to reflect the data it holds.